src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java

```diff
@@ -153,6 +153,20 @@ public void should_record_validation_failure_when_ocr_data_parsing_fails() throw
         errorWasSent(SAMPLE_ZIP_FILE_NAME, ErrorCode.ERR_METAFILE_INVALID);
     }
 
+    @Test
+    public void should_record_validation_failure_when_zip_does_not_contain_ocr_data() throws Exception {
+        // given
+        uploadToBlobStorage(SAMPLE_ZIP_FILE_NAME, zipDir("zipcontents/missing_ocr_data"));
```

Comment:
imho worth making it clearer as it sounds as if ocr data was always required.

Reply:
Do you mean rename the test name to make it clearer?

Reply:
is the scenario- new application with no scannable item with ocr data ?
`should_record_validation_failure_when_classification_is_new_app_and_no_scannable_item_includes_ocr`

Reply:
No. It is for new_application but checking ocr data only for SSCS1 type